File: for_loop.py
offices = [
    ['john', 'sam'],
    ['peter','stu', 'miles', 'isaac'],
    ['martha', 'chloe', 'zanthia'],
]
for office in offices:
    print(str(len(office)) + ': ', end='')
    for person in office:
        print(person, end=' ')
    print()

File: lua/for_loop.lua
offices = {
    {"john", "sam"},
    {"peter", "stu", "miles", "isaac"},
    {"martha", "chloe", "zanthia"}
}

for _, office in ipairs(offices) do
    io.write(#office .. ": ")
    for _, person in ipairs(office) do
        io.write(person .. " ")
    end
    io.write("\n")
end
